acmw-db-app/pages/signin.js

```diff
@@ -0,0 +1,129 @@
+import Head from 'next/head'
+import styles from '../styles/SignIn.module.css'
+import React from 'react';
+
+function SignIn() {
+  // TODO link to page that sends password by email?
```

Comment:
This was something I learned over the summer by avoiding using the "function" declaration and doing this instead:
`const SignIn = () => { // function stuff here
}`
This seems more of a preference thing but here's a better article on it: https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/

Reply:
Sounds good I'll try to remember that for the future! I'll try to change it soon and hopefully all the `this` stuff won't be hard to change.